proposals/0295-codable-synthesis-for-enums-with-associated-values.md

```diff
@@ -64,36 +65,37 @@ The top-level container contains a single key that matches the name of the enum
 which points to another container that contains the values as they would be encoded
 for structs and classes.
 
-Associated values can also be unlabeled, in which case they will be encoded into an
-array instead, or just the raw value if only one parameter is present.
+Associated values can also be unlabeled, in which case an identifier will be generated in the form of `_$N`, where `$N` is the 0-based position of the parameter.
```

Comment:
I think that it may be nice to add a note on how this solves a problem with changes to the shape of the `enum`.  Is there ay plan on handling something like this:

```swift
enum E {
case c(_1: String, Int)
}
```
This I believe would cause conflicts with `_1` no?  Is this treated as an overloaded case and simply not synthesized?

Reply:
Good point. This should be fine as long as there are no collisions, but if there are, we should emit a diagnostic. Right now it will just generate invalid code and then choke on that. So at least it rejects it, but without a useful error message. I'll add that.